pkg/matcher/annotation_matcher.go

```diff
@@ -242,7 +242,7 @@ func buildAvailableMatchingAnnotationErr(event *info.Event, pruns []*tektonv1.Pi
 			}
 			errmsg += fmt.Sprintf(" %s: ", strings.Replace(annotation, pipelinesascode.GroupName+"/", "", 1))
 			if annotation == keys.OnCelExpression {
-				errmsg += "celexpression"
+				errmsg += prun.GetAnnotations()[keys.OnCelExpression]
```

Comment:
we dont need this if condition even then

Reply:
:thinking: Not sure i understand completely
 If there is a cel annotation than only we need to add that value to errmsg otherwise not 
Or you saying something different ?

Reply:
I am saying like the current code

```
for annotation, value := range prun.GetAnnotations() {
	if !strings.HasPrefix(annotation, pipelinesascode.GroupName+"/on-") {
		continue
	}
	errmsg += fmt.Sprintf(" %s: ", strings.Replace(annotation, pipelinesascode.GroupName+"/", "", 1))
	if annotation == keys.OnCelExpression {
		errmsg += prun.GetAnnotations()[keys.OnCelExpression]
	} else {
		errmsg += value
	}
}
```
is same as 

```
for annotation, value := range prun.GetAnnotations() {
	if !strings.HasPrefix(annotation, pipelinesascode.GroupName+"/on-") {
		continue
	}
	errmsg += fmt.Sprintf(" %s: %s", strings.Replace(annotation, pipelinesascode.GroupName+"/", "", 1), value)
}
```

Reply:
Updated :+1: